### PR TITLE
Update Chinese Translation

### LINKED ISSUE #26295: Incomplete Chinese Translation for Login Page
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

translations

### Describe the bug

Chinese Translation are out of date, and some translation key are missing from chinese trasnaltion file.

### Version

23.0.4

### Expected behavior

Most of strings should have its own correct translation

### Actual behavior

Fallbacked to the english translation.

### How to Reproduce?

1. Switch the language to chinese.
2. Trigger password error, and the error will be showed in english.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by hyprocritor**
Added the missing part of chinese translations and correct some wrong translations.

Closes #26295

### CODE REVIEW

**Comment by jonkoops** [comment]
@xianli123 could you review this one?

**Comment by xianli123** [comment]
@czk1997 Thanks for your contribution. Looks very great. Just added 2 comments in the file, please take a look.

**Comment by czk1997** [comment]
> @czk1997 Thanks for your contribution. Looks very great. Just added 2 comments in the file, please take a look.

I am afraid I did not find any comments in the file.

**Comment by xianli123** [comment]
Sorry @czk1997 , I forgot to submit my comments. Could you see the comments now?

File: themes/src/main/resources-community/theme/base/login/messages/messages_zh_CN.properties [hunk 1]
```diff
@@ -84,22 +167,29 @@ emailLinkIdpTitle=链接 {0}
...
-emailInstruction=输入您的用户名和邮箱，我们会发送一封带有设置新密码步骤的邮件到您的邮箱。
+emailInstruction=输入您的用户名或邮箱，我们会发送一封带有设置新密码步骤的邮件到您的邮箱。
+emailInstructionUsername=输入您的用户名，我们将向您发送如何创建新密码的说明。
+
+copyCodeInstruction=请复制这段验证码并粘贴到您的应用：
 
-copyCodeInstruction=请复制这段验证码并粘贴到应用：
+pageExpiredTitle=页面已过期
+pageExpiredMsg1=来重启登录流程
+pageExpiredMsg2=来继续登录流程
```
[the remark this answers is not in the document]
Comment (czk1997) [reply themes/src/main/resources-community/theme/base/login/messages/messages_zh_CN.properties]: fixed.

File: themes/src/main/resources-community/theme/base/login/messages/messages_zh_CN.properties [hunk 2]
```diff
@@ -110,107 +200,322 @@ role_manage-clients=管理客户
...
+saml.artifactResolutionServiceInvalidResponse=无法解析构造。
+
+# Authenticators
+otp-display-name=验证器应用程序
+otp-help-text=从验证器应用输入验证码。
+otp-reset-description=应删除哪个OTP配置？
+password-display-name=密码
+password-help-text=通过输入密码登录。
+auth-username-form-display-name=用户名
+auth-username-form-help-text=输入用户名已开始登录
```
[the remark this answers is not in the document]
Comment (czk1997) [reply themes/src/main/resources-community/theme/base/login/messages/messages_zh_CN.properties]: fixed.

**Comment by czk1997** [comment]
@xianli123 I had fixed the issue. Please check.

**Review by xianli123 — APPROVED** [review]

**Comment by xianli123** [comment]
Thanks @czk1997. It looks great.

**Review by jonkoops — APPROVED** [review]
Thanks for the contribution @czk1997, I will set this to merge.

**Review by ahus1 — APPROVED** [review]
Approving as of @jonkoops's review.

**Pull request merged**